worlds/messenger/Shop.py

```diff
@@ -0,0 +1,102 @@
+import random
+from random import Random
+from typing import Dict, TYPE_CHECKING, NamedTuple, List, Tuple
+
+if TYPE_CHECKING:
+    from . import MessengerWorld
+else:
+    MessengerWorld = object
+
+PROG_SHOP_ITEMS = [
+    "Path of Resilience",
+    "Meditation",
+    "Strike of the Ninja",
+    "Second Wind",
+    "Currents Master",
+    "Aerobatics Warrior",
+]
+
+USEFUL_SHOP_ITEMS = [
+    "Karuta Plates",
+    "Serendipitous Bodies",
+    "Kusari Jacket",
+    "Energy Shuriken",
+    "Serendipitous Minds",
+    "Rejuvenate Spirit",
+    "Demon's Bane",
+]
+
+
+class ShopData(NamedTuple):
+    internal_name: str
+    min_price: int
+    max_price: int
+    default_price: int = 0
```

Comment:
this is mostly for documentation purposes and gives a quick reference point since the randomized ranges are mostly guess work for what "feels good" while being similar to the original price